libpod/sql_state_test.go

```diff
@@ -102,7 +111,7 @@ func getEmptyState() (s State, p string, err error) {
 	dbPath := filepath.Join(tmpDir, "db.sql")
 	lockPath := filepath.Join(tmpDir, "db.lck")
 
-	state, err := NewSQLState(dbPath, lockPath, tmpDir, nil)
+	state, err := NewSQLState(dbPath, lockPath, tmpDir, tmpDir, nil)
```

Comment:
'nother nit.  You'd never have your specdir and lockdir be the same right?  I woudln't test that way.